wallet/chainntfns.go

```diff
@@ -885,6 +886,15 @@ func (w *Wallet) VoteOnOwnedTickets(winningTicketHashes []*chainhash.Hash,
 			return n.PublishTransaction(context.TODO(), vote)
 		})
 		if err != nil {
+			ruleErr, ok := err.(dcrjson.RPCError)
+			if ok {
+				if ruleErr.Code == dcrjson.ErrRPCDuplicateTx {
```

Comment:
should be != since we don't want to log the error in this case.  it should also still run the info log that follows this error check so there is a log message that a vote was created and the wallet still did its job correctly.